tdrs-backend/tdpservice/parsers/test/test_parse.py

```diff
@@ -498,9 +514,9 @@ def test_parse_ssp_section1_datafile(ssp_section1_datafile):
     ssp_section1_datafile.year = 2019
     ssp_section1_datafile.quarter = 'Q1'
 
-    expected_m1_record_count = 7849
-    expected_m2_record_count = 9373
-    expected_m3_record_count = 16764
+    expected_m1_record_count = 820
+    expected_m2_record_count = 992
+    expected_m3_record_count = 1757
```

Comment:
Is the dramatic change in record counts due to the fact that this particular file has records with a RPT_MONTH_YEAR spanning from 2018 - 2021?

Reply:
exactly, and since this is cat1 error it will not parse the fields and subsequently less errors